test/helpers/set-config.js

```diff
@@ -0,0 +1,63 @@
+function setProjectConfigForErrors(project) {
```

Comment:
FWIW, I'd rather not even have these helper methods, and just use `project` methods to setup each test as appropriate. I find the "DRY" ness here actually makes each individual test harder to read and understand.

I know you didn't introduce these helpers, sorry for making extra work. 😩

Reply:
Would you like to have to have them in `fake-project` module, `project.setProjectConfigForErrors` for instance? Or do you prefer to inline them in every test?